wrapper/minecraft/src/main/kotlin/org/quiltmc/qkl/wrapper/minecraft/nbt/NbtConversions.kt

```diff
@@ -0,0 +1,322 @@
+/*
+ * Copyright 2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qkl.wrapper.minecraft.nbt
+
+import net.minecraft.nbt.*
+
+//region: NBT <-> Raw values
```

Comment:
A possible concern with these extensions is global namespace pollution. Is this a common enough use-case to warrant .nbt popping up on every primitive?

Reply:
My 2c is that `nbt` isnt a common enough thing to begin typing for it too matter much unless you want this, but thats my thoughts. The auto complete is a confusing and fickle beast